hack/Dockerfile.markdownlint

```diff
@@ -1,4 +1,4 @@
-FROM centos
+FROM fedora:29
```

Comment:
not ideal since fedora 29 isn't maintained anymore, but I know you said you saw this used elsewhere.  We should come back around and change it again at some point though ...

Reply:
Yeah, I wanted 33, but that wasn't available in CI and I thought it would be better to be consistent.